### PR TITLE
deadlines.html

### PR DESCRIPTION
**Pull request by cchan04**

### CODE REVIEW

File: deadlines.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<head>
+
+    <link rel="stylesheet" type="text/css" href="css/about.css"/>
+    <script type="text/javascript" src="js/about.js"></script>
```
Comment (adflynn) [inline deadlines.html]: why are you linking to the about.css! This should link to the deadlines.css!

File: deadlines.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+         <h2> Search Colleges </h2>
+      </div>
+
+      <div class="quiz section">
+          <h2>Find Your Perfect College Fit!</h2>
+      </div>
+
+      <div class="calendar section">
+      <h2>
+        <a href="./deadlines.html">Deadlines Calendar</a>
```
Comment (adflynn) [inline deadlines.html:39]: nice !!

File: deadlines.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      </h2>
+    </div>
+      <div class="about us section">
+        <h2>
+        <a href="./about.html">About Us!</a>
+         </h2>
+      </div>
+
+      <div class="scholarship section">
+          <h2>Scholarship Info</h2>
```
Comment (AdrianaChavez) [inline deadlines.html:26]: I changed the div to "scholarships" :)

File: deadlines.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      <div class="calendar section">
+      <h2>
+        <a href="./deadlines.html">Deadlines Calendar</a>
+      </h2>
+      </div>
+  </div>
+
+    <div class="info">
+      <p>
+        A place to go to feel organized. Look into the future, plan ahead.
```
Comment (AdrianaChavez) [inline deadlines.html:46]: Great line!!

File: deadlines.html (hunk 2 above)
Comment (hemasphere) [inline deadlines.html]: add end tag

Comment (hemasphere) [reply deadlines.html:39]: looks good

File: deadlines.html (hunk 3 above)
Comment (hemasphere) [reply deadlines.html:26]: looks good

File: deadlines.html (hunk 4 above)
Comment (hemasphere) [reply deadlines.html:46]: looks good

**Pull request merged**